packages/dd-trace/src/noop/proxy.js

```diff
@@ -1,10 +1,10 @@
 'use strict'
 
 const NoopTracer = require('./tracer')
-const NoopAppsecSdk = require('../appsec/sdk/noop')
+const NoopAppsecSDK = require('../appsec/sdk/noop')
```

Comment:
If you search in dd-trace, there is zero occurence for `SDK` (except comments) and 26 occurences of `Sdk`, so I'd say let's keep it consistent ?